tools/refguide_check.py

```diff
@@ -205,7 +205,7 @@ def find_names(module, names_dict):
             res = re.match(pattern, line)
             if res is not None:
                 name = res.group(1)
-                entry = '.'.join([module_name, name])
+                ## entry = '.'.join([module_name, name])
```

Comment:
it looks like this line can just be removed?

Reply:
Indeed, I commented it to please the linter.